src/components/job-overview/style.css

```diff
@@ -1,25 +1,3 @@
-.job-overview__breadcrumbs {
-  margin-bottom: var(--eds_spacing_xx_large);
-}
-.job-overview__btn.job-overview__btn-stop,
-.job-overview__btn.job-overview__btn-stop:disabled {
-  margin: var(--eds_spacing_small) 0;
-}
-.job-overview__overview-content {
-  display: grid;
-  gap: 0 var(--eds_spacing_medium);
-  grid-template-columns: auto auto;
-}
-.job-overview__overview-content p,
-.job-overview__artefacts-content p {
-  line-height: var(--eds_spacing_large);
-}
-.job-overview__overview,
-.job-overview__artefacts,
-.job-overview__stepslist {
-  margin: var(--eds_spacing_x_large) 0;
-}
-
 .steps-list__content {
```

Comment:
this can be removed if using grid spacing

Reply:
removed